File: app/src/main/java/wacode/yamada/yuki/nempaymentapp/view/activity/DeepLinkActivity.java
package wacode.yamada.yuki.nempaymentapp.view.activity;

import android.app.Activity;
import android.os.Bundle;

import com.airbnb.deeplinkdispatch.DeepLinkHandler;

import wacode.yamada.yuki.nempaymentapp.view.activity.deeplink.LibraryDeepLinkModuleLoader;
import wacode.yamada.yuki.nempaymentapp.view.activity.deeplink.SimpleModule;
import wacode.yamada.yuki.nempaymentapp.view.activity.deeplink.LibraryDeepLinkModule;
import wacode.yamada.yuki.nempaymentapp.view.activity.deeplink.SimpleModuleLoader;

@DeepLinkHandler({ SimpleModule.class, LibraryDeepLinkModule.class })
public class DeepLinkActivity extends Activity {
    @Override protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        DeepLinkDelegate deepLinkDelegate =
                new DeepLinkDelegate(new SimpleModuleLoader(), new LibraryDeepLinkModuleLoader());
        deepLinkDelegate.dispatchFrom(this);
        finish();
    }
}

File: app/src/main/java/wacode/yamada/yuki/nempaymentapp/view/activity/DeepLinkActivity.kt
package wacode.yamada.yuki.nempaymentapp.view.activity

import android.app.Activity
import android.os.Bundle
import com.airbnb.deeplinkdispatch.DeepLinkHandler
import wacode.yamada.yuki.nempaymentapp.view.activity.deeplink.LibraryDeepLinkModule
import wacode.yamada.yuki.nempaymentapp.view.activity.deeplink.LibraryDeepLinkModuleLoader
import wacode.yamada.yuki.nempaymentapp.view.activity.deeplink.SimpleModule
import wacode.yamada.yuki.nempaymentapp.view.activity.deeplink.SimpleModuleLoader

@DeepLinkHandler(SimpleModule::class, LibraryDeepLinkModule::class)
class DeepLinkActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val deepLinkDelegate = DeepLinkDelegate(SimpleModuleLoader(), LibraryDeepLinkModuleLoader())
        deepLinkDelegate.dispatchFrom(this)
        finish()
    }
}
